cli/commands/create_submission_pipeline_command.py

```diff
@@ -0,0 +1,176 @@
+import os
+import sys
+import importlib
+import humps
+from typing import Optional
+from configargparse import ArgParser
+from pathlib import Path
+from string import Template
+
+from mcs_benchmark_data.dataset_type import DatasetType
+from mcs_benchmark_data.template_type import TemplateType
+
+from cli.commands._command import _Command
+
+
+class CreateSubmissionPipelineCommand(_Command):
+    def add_arguments(self, arg_parser: ArgParser, add_parent_args):
+        arg_parser.add_argument(
+            "--benchmark_name",

```

Comment:
`--benchmark-name`. Command line parameters are kebab-case. They're translated to snake_case variables on `.args` by the ArgParser machinery.